client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -46,9 +53,14 @@ const ActivityLogV2: FunctionComponent = () => {
 		( state ) => siteId && siteHasFeature( state, siteId, WPCOM_FEATURES_FULL_ACTIVITY_LOG )
 	);
 	const settingsUrl = useSelector( ( state ) => getSettingsUrl( state, siteId, 'general' ) );
+	let upgradePlanNames = translate( 'Jetpack Personal or Jetpack Premium' );
```

Comment:
For a Jetpack site, I see this upsell message with these changes:

<img width="696" alt="Screen Shot 2024-05-22 at 11 30 23 AM" src="https://github.com/Automattic/wp-calypso/assets/1689238/ad9d7f89-f4b7-4e7a-a0e2-595ad6b0155c">

Without these changes, I see:

<img width="697" alt="Screen Shot 2024-05-22 at 11 30 32 AM" src="https://github.com/Automattic/wp-calypso/assets/1689238/dc4aa5a5-347d-46bd-98e3-c0bba66584a8">

Unless Jetpack requested that change, we should keep it the same as it is now. @candy02058912 @lupus2k

Reply:
Oh, it also looks like there is a Starter plan upsell beneath that that shouldn't be there on Jetpack sites.
<img width="710" alt="Screenshot 2024-05-22 at 10 56 53 AM" src="https://github.com/Automattic/wp-calypso/assets/140841/4dde9613-ac66-4215-ba91-f5aa73e93097">

Reply:
> Unless Jetpack requested that change, we should keep it the same as it is now.

Reverted changes to the Activity Log top large upsell: https://github.com/Automattic/wp-calypso/pull/89508/commits/8796fb9380f3f76100499ecd66c351fcf6da8365

Reply:
> Oh, it also looks like there is a Starter plan upsell beneath that that shouldn't be there on Jetpack sites.

Limited it to only show for simple sites: https://github.com/Automattic/wp-calypso/pull/89508/commits/e70f3b9b5e88fd3d74a16643399c22a984a5700c